src/components/cms/SkillFeedbackPage/EditFeedbackDialog.js

```diff
@@ -8,56 +8,64 @@ import FormControl from '@material-ui/core/FormControl';
 import FormHelperText from '@material-ui/core/FormHelperText';
 import Button from '../../shared/Button';
 
-const EditFeedback = props => {
-  const {
-    handleConfirm,
-    handleClose,
-    feedback,
-    errorText,
-    handleEditFeedback,
-  } = props;
-  return (
-    <React.Fragment>
-      <DialogTitle>Edit Feedback</DialogTitle>
-      <DialogContent>
-        <FormControl fullWidth={true}>
-          <OutlinedTextField
-            id="edit-feedback"
-            label="Skill Feedback"
-            placeholder="Skill Feedback"
-            defaultValue={feedback || ''}
-            multiLine={true}
-            fullWidth={true}
-            onChange={handleEditFeedback}
-            aria-describedby="edit-feedback-helper-text"
-          />
-          <FormHelperText id="edit-feedback-helper-text" error>
-            {errorText}
-          </FormHelperText>
-        </FormControl>
-      </DialogContent>
-      <DialogActions>
-        <Button
-          key={0}
-          variant="contained"
-          color="primary"
-          onClick={handleClose}
-        >
-          Cancel
-        </Button>
-        <Button
-          key={1}
-          variant="contained"
-          color="primary"
-          onClick={handleConfirm}
-          style={{ marginRight: '1em' }}
-        >
-          Edit
-        </Button>
-      </DialogActions>
-    </React.Fragment>
-  );
-};
+class EditFeedback extends Component {
+  state = {
+    errorText: '',
+  };
+  submitForm = () => {
+    let feedbackText = document.getElementById('edit-feedback').value;
+    if (!feedbackText) {
+      this.setState({ errorText: 'Feedback cannot be empty' });
+    } else {
+      this.props.handleConfirm();
+    }
+  };
+  render() {
+    const { handleClose, feedback, handleEditFeedback } = this.props;
+    const { errorText } = this.state;
+    return (
+      <React.Fragment>
+        <DialogTitle>Edit Feedback</DialogTitle>
+        <DialogContent>
+          <FormControl fullWidth={true}>
+            <OutlinedTextField
+              id="edit-feedback"
```

Comment:
Remove this logic of getting value by `id`. Instead declare a state variable for it.